{{cookiecutter.repo_slug}}/.github/labels.yml

```diff
@@ -25,41 +25,41 @@
   description: "Generic maintenance tasks."
 - name: "ci"
   color: 1d76db
-  description: "Work that improves the continue integration."
+  description: "Work that improves the continuous integration."
 - name: "dependencies"
   color: 1d76db
-  description: "Upgrade or downgrade of project dependencies."
+  description: "Change in project dependencies."
 
 - name: "in-progress"
   color: fbca04
-  description: "Issue is currently being resolved by a developer."
+  description: "Issue is currently being worked on by a developer."
 - name: "stale"
   color: fef2c0
-  description: "There has not been activity on this issue or PR for quite some time."
+  description: "No activity for quite some time."
 - name: "no-stale"
   color: fef2c0
-  description: "This issue or PR is exempted from the stable bot."
+  description: "This is exempt from the stale bot."
 
 - name: "security"
   color: ee0701
-  description: "Marks a security issue that needs to be resolved ASAP."
+  description: "Addressing a vulnerability or security risk in this project."
 - name: "incomplete"
   color: fef2c0
-  description: "Marks a PR or issue that is missing information."
+  description: "Missing information."
 - name: "invalid"
   color: fef2c0
-  description: "Marks a PR or issue that is missing information."
+  description: "This is off-topic, spam, or otherwise doesn't apply to this project."
```

Comment:
The description here was incorrect (copied from "incomplete"), so this is my suggestion.